### PR TITLE
Added Cypress test framework

### PR DESCRIPTION
**Pull request by jonparker**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -154,6 +154,10 @@ git push heroku master
 
 * This also runs all of the build and deployment commands, so it may take a moment, but once it's complete, the live link should be updated.
 
+## Running tests with Cypress.io
+
+Run `npx cypress open`
```
Comment (qdozaq) [inline README.md:159]: Should probably just go ahead and put this behind an `npm` script.
something like
```
$ npm run test:e2e
```

Comment (azmainamin) [reply README.md:159]: +++

**Comment by azmainamin** [comment]
@jonparker Is the work still pending?

**Comment by jonparker** [comment]
I've not done any further work on this.
Feel free to take any learning and document/share and close this PR.

**Pull request closed without merging**